packages/analytics/analytics-chart/src/components/TopNTable.vue

```diff
@@ -234,9 +234,8 @@ const getValue = (record: AnalyticsExploreRecord): string => {
 
 .kong-ui-public-top-n-table {
   border: none !important;
-  height: 100%;
+  max-height: 100%;
```

Comment:
This seems like a meaningful change.  Isn't this going to break TopNTable styling in dashboards?

Reply:
Double-checked; looks good.